BraintreeCore/src/main/java/com/braintreepayments/api/core/UserCanceledException.kt

```diff
@@ -14,7 +14,7 @@ import androidx.annotation.RestrictTo
  * could mean the user returned to the app through multi-tasking without completing the flow,
  * the user closed the browser tab, or the user pressed the back button.
  */
-open class UserCanceledException @JvmOverloads @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) constructor(
+class UserCanceledException @JvmOverloads @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP) constructor(
     message: String?,
-    val isExplicitCancelation: Boolean = false
+    val isExplicitCancellation: Boolean = false
```

Comment:
Maybe a separate PR but do we even need this property anymore? I think we only return `UserCanceledException` for explicit cancellations in v5 right?

Reply:
Ah good callout in v5 we `:kill_it_with_fire:` this is the best option lol.

Reply:
I'll remove this property in a follow up PR 👍